Source/Particles/Collision/BinaryCollision/NuclearFusion/ProtonBoronFusionCrossSection.H

```diff
@@ -71,7 +71,7 @@ amrex::ParticleReal ProtonBoronFusionCrossSectionNevins (const amrex::ParticleRe
         constexpr auto D2 = -20.3_prt;
         constexpr auto D5 = -1.58_prt;
         const amrex::ParticleReal E_norm = (E_keV-400._prt) * 1.e-2_prt;
-        astrophysical_factor = D0 + D1*E_norm + D2*E_norm*E_norm + D5*std::pow(E_norm,5);
+        astrophysical_factor = D0 + D1*E_norm + D2*E_norm*E_norm + D5*std::pow(E_norm,5._prt);
```

Comment:
A bit surprising this warns, should be fine with
```suggestion
        astrophysical_factor = D0 + D1*E_norm + D2*E_norm*E_norm + D5*std::pow(E_norm, 5);
```
https://godbolt.org/z/ezx75EzKa

Reply:
@dpgrote just to confirm, does using `int`s here still create a warning for you?
Can you please double check before we merge? :)

Reply:
Yes, using `int` here is what triggered the warning in the first place.

Reply:
Very unusual. Is that only on macOS?

Reply:
More details:
```console
mpicxx --version
g++ (Homebrew GCC 11.2.0_3) 11.2.0
```
```console
make -j 16 PRECISION=FLOAT USE_SINGLE_PRECISION_PARTICLES=TRUE
```
```
./Source/Particles/Collision/BinaryCollision/NuclearFusion/ProtonBoronFusionCrossSection.H:74:66: warning: conversion from '__gnu_cxx::__promote_2<float, int, float, double>::__type' {aka 'double'} to 'amrex::ParticleReal' {aka 'float'} may change value [-Wfloat-conversion]
   74 |         astrophysical_factor = D0 + D1*E_norm + D2*E_norm*E_norm + D5*std::pow(E_norm,5);
      |                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~
```

https://stackoverflow.com/questions/5627030/why-was-stdpowdouble-int-removed-from-c11
```
The only C++03 signature that has changed is:
  float pow(float, int);
This now returns double:
  double pow(float, int);
And this change was done for C compatibility.
```

Reply:
We agree to follow a random stack overflow person (see link above) that says:
> So while the freedom is there to shuffle pow(double, int) off to a separate algorithm, most implementors I'm aware of have given up on that strategy, with the possible exception of checking for very small integral exponents. And in that event, it is usually advantageous to put that check in the implementation with the floating point exponent so as to get the biggest bang for your optimization buck.

:grin: 

So float exponent it is :D